### DISCUSSION TITLE
[This Is A Test] Welcome to SIG-Testing Discussions!

### DISCUSSION THREAD

**Discussion by BenTheElder** [kubernetes/sig-testing#1] [Announcements]
## 👋 Welcome!

We are testing GitHub Discussions as an option for asynchronous communication within [SIG Testing](https://github.com/kubernetes/community/blob/master/sig-testing/README.md).

This page is a test 🙂 
Let's see how well this works! 😁

**Comment by nikhita** [comment]
:wave: Hello!

**Reply by BenTheElder** [reply]
Hello!

I'm not sure about the voting up/down of comments sorting them on the page versus just 👍  etc. 🤔 

We will have to experiment with this more though 😅

**Reply by BenTheElder** [reply]
It seems that it defaults to sort by votes ("top") for Q & A which makes sense actually.

Switching the discussion type to "show and tell" sorts by oldest by default (and has options for newest, top).

**Comment by spiffxp** [comment]
/meow
wonder if prow listens to these

**Reply by spiffxp** [reply]
it does not (probably fine, we'd have to figure out which thread to reply to)

**Reply by spiffxp** [reply]
a link to this discussion: https://github.com/kubernetes/sig-testing/discussions/1

**Reply by hh** [reply]
> it does not (probably fine, we'd have to figure out which thread to reply to)

Sub-reploy

**Comment by spiffxp** [comment]
How do we mark a discussion as `answered` vs. `unanswered`?

**Reply by spiffxp** [reply]
Changed the category of this discussion to Q&A, and saw this
![Screen Shot 2021-02-23 at 1 33 27 PM](https://user-images.githubusercontent.com/49258/108890547-b7fc8780-75db-11eb-88bc-69dfaa2d7720.png)

**Reply by spiffxp** [reply]
Can make new categories (as a repo admin?)
![Screen Shot 2021-02-23 at 1 34 03 PM](https://user-images.githubusercontent.com/49258/108890666-d9f60a00-75db-11eb-9c1d-dec546e1b146.png)

**Reply by BenTheElder** [reply]
Changing to Q&A also changes the sorting to be by voting, whereas this is not the case for other categories https://github.com/kubernetes/sig-testing/discussions/1#discussioncomment-398721

**Comment by hh** [comment]
Bots might need minor changes to reach and react!

**Reply by hh** [reply]
Looks like there is top level, and second level, no third level threads.

**Reply by hh** [reply]
Testing for third.

**Comment by aojea** [comment]
:)

**Reply by aojea** [reply]
XD

**Reply by aojea** [reply]
;)

**Comment by matthyx** [comment]
Hi Ben :)

**Reply by BenTheElder** [reply]
Hi!

**Comment by tehcyx** [comment]
🎈  to the top

**Reply by spiffxp** [reply]
Bump
